packages.csv

```diff
@@ -11,6 +11,7 @@ Ubuntu-24.04,azcopyArmDeb,microsoft-ubuntu-noble-prod-apt,noble
 Debian-9.0,azcopyAmdDeb,microsoft-debian-stretch-prod-apt,stretch
 Debian-10.0,azcopyAmdDeb,microsoft-debian-buster-prod-apt,buster
 Debian-11.0,azcopyAmdDeb,microsoft-debian-bullseye-prod-apt,bullseye
+Debian-12.0,azcopyAmdDeb,microsoft-debian-bookworm-prod-apt,bookworm
```

Comment:
There was a similar issue in Blobfuse asking for Debian 12 support, https://github.com/Azure/azure-storage-fuse/issues/1510

We can ask the customer to try this,
```
RUN ID=$( grep "^ID=" /etc/os-release | cut -d '=' -f 2 | tr -d '"') && \
    VERSION_ID=$(grep "^VERSION_ID=" /etc/os-release | cut -d '=' -f 2 | tr -d '"') && \
    VERSION_ID=11 && \
    URL="https://packages.microsoft.com/config/${ID}/${VERSION_ID}/packages-microsoft-prod.deb" && \
    curl $URL -o /tmp/packages-microsoft-prod.deb && \
    dpkg -i /tmp/packages-microsoft-prod.deb && \
    apt-get update && \
    apt-get install -y libfuse3-dev fuse3 blobfuse2 && \
    rm /tmp/packages-microsoft-prod.deb 
```

This step is basically downloading the `packages-microsoft-prod.deb` from Debian 11 and then installing blobfuse.
Reason for saying this is we should not directly release in Debian 12 without proper testing. We can discuss this more in our team's meeting.

Reply:
ohh is it. I was not aware of this. Sure, lets discuss this in the team meeting.

Reply:
I have added test for debian-12. Please check.